templates/InstantSearch.js/.template.js

```diff
@@ -0,0 +1,25 @@
+const build = require('../../packages/tasks/common/build');
+const clean = require('../../packages/tasks/common/clean');
+const install = require('../../packages/tasks/node/install');
+const teardown = require('../../packages/tasks/node/teardown');
+
+module.exports = {
+  libraryName: 'instantsearch.js',
+  tasks: {
+    setup: () => Promise.resolve(),
+    build,
+    install,
+    clean,
+    teardown,
+  },
+  keywords: [
```

Comment:
What are the keywords for?

Reply:
When compiling the template to demo, to add the keywords to the `package.json` (to get a result close to [the original template](https://github.com/algolia/instantsearch-templates/blob/master/src/InstantSearch.js/package.json#L5)).